src/observation.c

```diff
@@ -218,7 +218,8 @@ u8 make_propagated_sdiffs(u8 n_local, navigation_measurement_t *m_local,
                             + dist_diff);
       sds[n].carrier_phase = m_local[i].carrier_phase
                            - (m_remote[j].carrier_phase
-                              - dist_diff / GPS_L1_LAMBDA);
+                             - dist_diff /
+                                (GPS_C / code_to_carr_freq(e[i]->sid.code)));
```

Comment:
I see this expression in multiple places. Does it worth to make a constant in the structure instead to avoid constant division?